example_app/attribute_generator.rb

```diff
@@ -32,7 +32,7 @@ def increment_random_attribute
   def next_random_attribute
     x = rand(100)
 
-    base_attributes.keys.find do |key|
+    base_attributes.keys.detect do |key|
```

Comment:
Unless this is an error in the script, it might be best to separate code changes from this already substantial structural change? If nothing else, it'll make for a cleaner history with regards to what PR made which change.

And out of personal curiosity - why change .find to .detect?

Reply:
I agree—moving the code changes to a separate PR makes the intent of both requests much clearer. What method would you propose we use to move the code changes out of this PR?

Regarding `find` vs. `detect`: That was to satisfy the linter, which is currently using Thoughtbot's config. I don't have a preference either way, so if you like `find`, let's use it and update the config.